ballerina/modules/parser/parser.bal

```diff
@@ -118,26 +118,33 @@ public class Parser {
             if token.kind != T_DOLLAR {
                 return getExpectedCharError(token, DOLLAR);
             }
-            Location location = token.location.clone();
+            Location varDefinitionLocation = token.location.clone();
             token = check self.readNextNonSeparatorToken();
-            string variableName = check getIdentifierTokenvalue(token);
+            string varName = check getIdentifierTokenvalue(token);
```

Comment:
Any specific reason to change this variable name?

Reply:
just to increase the readability by making variables use to create  `VariableDefinition` start with prefix `var`.